extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -234,60 +240,61 @@ export class LinkerManager {
   }
 
   /**
-   * Called on click on any anchor element. Adds linker param if a match for
-   * given linker configuration.
-   * @param {!Element} element
-   * @param {!Event} event
+   * Apply linkers to the given url.
```

Comment:
Should we clarify that we might not append the linker params when this is called?

Reply:
done